<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChargeDialog</class>
 <widget class="QDialog" name="ChargeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>105</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lbMoney">
       <property name="minimumSize">
        <size>
         <width>54</width>
         <height>12</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>54</width>
         <height>12</height>
        </size>
       </property>
       <property name="text">
        <string>余额</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leMoney">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lbCharge">
       <property name="minimumSize">
        <size>
         <width>54</width>
         <height>12</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>54</width>
         <height>12</height>
        </size>
       </property>
       <property name="text">
        <string>充值金额</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leCharge">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnConfirm">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
